core/server/api/canary/utils/serializers/input/members.js

```diff
@@ -64,6 +77,11 @@ module.exports = {
                 }
             });
         }
+
+        if (frame.data.members[0].tiers) {
+            frame.data.members[0].products = frame.data.members[0].tiers;
```

Comment:
Does this mapping need to be applied to the edit endpoint as well?

Reply:
The `edit` serializer defers the logic to `add` [here](https://github.com/TryGhost/Ghost/blob/2e8c66d93c4abdba485fa280790312f1ee82f142/core/server/api/canary/utils/serializers/input/members.js#L72), so it automatically gets applied to it :)